graphql/src/test/java/com/scalar/db/graphql/schema/BigIntCoercingTest.java

```diff
@@ -0,0 +1,140 @@
+package com.scalar.db.graphql.schema;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.scalar.db.io.BigIntValue;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.junit.Test;
+
+public class BigIntCoercingTest {
+  @Test
+  public void serialize_LongInValidRangeGiven_ShouldReturnLong() {
+    // Arrange
+    Long value = BigIntValue.MAX_VALUE;
+    Long expected = BigIntValue.MAX_VALUE;
+
+    // Act
+    Long actual = BigIntCoercing.INSTANCE.serialize(value);
+
+    // Assert
+    assertThat(actual).isEqualTo(expected);
+  }
+
+  @Test
+  public void serialize_LongOutOfRangeGiven_ShouldThrowException() {
+    // Arrange
+    Long value = BigIntValue.MAX_VALUE + 1;
```

Comment:
Can you also add a test that checks the other side (MIN_VALUE -1) of the range as well?

Reply:
Thank you! Added some tests in fd06d00.